lib/builder-webpack4/src/index.ts

```diff
@@ -71,6 +71,7 @@ export const start: WebpackBuilder['start'] = async ({ startTime, options, route
     publicPath: config.output?.publicPath as string,
     writeToDisk: true,
     logLevel: 'error',
+    watchOptions: config.watchOptions || {},
```

Comment:
Does this also need to be updated in `builder-webpack5` for for consistency?

Reply:
Does `builder-webpack5` load `webpack-dev-middleware` v3 or v4?

Reply:
Aha, it loads v4. Great! 🙏

Reply:
watchOptions was actually removed in `webpack-dev-middleware@4` so this breaks if v4 is used:

https://github.com/webpack/webpack-dev-middleware/blob/master/CHANGELOG.md#breaking-changes

Reply:
under what conditions would `webpack-dev-middleware@4` be used with `webpack@4`?

Reply:
The options are also passed below in core-server/src/manager/builder.ts, I came across the error when trying the instructions here: https://gist.github.com/shilman/8856ea1786dcd247139b47b270912324#gistcomment-3707867

Reply:
Gotcha jqrun. We might rework this in 6.3, which would be a workaround https://github.com/storybookjs/storybook/issues/14044